proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,235 @@
+
+  // Supplier Name
+  string supplier_code_name = 11;
+
+  // product_code
+  cmp.types.v1alpha.ProductCode product_code = 12;
+
+  // Duration range (min, max) of the activity in minutes
+  cmp.types.v1alpha.DurationRange duration_range = 13;
+
+  // Time given for confirmation before the offer expires
+  cmp.types.v1alpha.Duration max_confirmation_duration = 14; 
+
+  // Allow Free Sale
+  bool allow_free_sale = 15;
+
+  // The ability to confirm instantly
+  bool instant_confirmation = 16;
+
+  // Can be delivered instantly
+  bool instant_delivery = 17;
+
+  // Check availability
+  bool availability_required = 18;
+  
+  // Availability Type
+  string availability_type = 19;
+
+  // Delivery Formats; such as QR Code, NFT, Ticket, ... etc
+  repeated string delivery_formats = 20;
+
+  // Delivery Methods; such as Email, SMS, Post, ... etc
+  repeated string delivery_methods = 21;
+
+  // Redemption method
+  string redemption_method = 22;
+
+}
+
+
+message ActivityUnitInfo {
+  // Schedule
+  cmp.types.v1alpha.DateTimeRange schedule = 1; 
+
+  // Description of the activity unit
+  string descriptive_text = 2;
+
+  // Unit Code
+  string unit_code = 3;
+
+  // Unit Code Description
+  string unit_code_description = 4;
+
+}
+
+
+message ActivityServiceInfo {
+  // Description of the Service Info
+  string descriptive_text = 1;
+  
+  // Service Code
+  string service_code = 2;
+
+  // Service Code Description
+  string service_code_description = 3;
+
+  // Included items in the activity service
+  repeated string included = 4;
+
+  // Encluded items in the activity service
+  repeated string excluded = 5;
+}
+
+message ActivityZoneInfo {
+  // Sub Zone Info
+  ActivitySubZoneInfo sub_zone_info = 1;
+
+  // Unit Code
+  string unit_code = 2;
+}
+
+message ActivitySubZoneInfo {
+  // Geo tree type, representted by Country, Region, and City_or_Resort.
+  repeated cmp.types.v1alpha.GeoTree ref_points = 1;
+
+  // operation time shows the pick-up and drop-off information
+  repeated OperationTime operation_time = 2;
+
+  // Unit Code
+  string unit_code = 3;
+}
+
+message OperationTime {
+  // pick-up and drop-off information about location and time
+  repeated cmp.services.activity.v1alpha.PickupDropoffEvent pickup_dropoff = 1;
+
+  // Unit Code
+  string unit_code = 2;
+}
+
+
+message ActivityFeature {
+  // Feature description
+  string descriptive_text = 1;
+
+  // Feature Code
+  string feature_code = 2; 
+}
+
+message ActivityTag {
+  // active status
+  bool active = 1;
+
+  // Unique ID
+  int32 id = 2;
+
+  // Name
+  string name = 3;
+
+  // Slug; giving an informal name to the tag
+  string slug = 4;
+}
+
+message ActivityContactInfo {
+
+  // Address
+  cmp.types.v1alpha.Address address = 1;
+
+  // Phones
+  repeated cmp.types.v1alpha.Phone phones = 2;
+
+  // Emails
+  repeated cmp.types.v1alpha.Email emails = 3;
+
+  // Websites
+  repeated cmp.types.v1alpha.Link links = 4;    
+
+}
```

Comment:
This looks like generic information. We should move this to `types` and name it as `ContactInfo`. Wdyt?

Reply:
I thought about that, but when I looked at accommodation, I found that they don't have a separate object for contact info and phone, email are part of a bigger message.

Reply:
Maybe we should create a types.ProductContactInfo, so it is clear that it is not for travellers